endpoint.rst

```diff
@@ -220,9 +216,26 @@ endpoint:
     $ sudo singularity remote remove --global <remote_name>
 
 
+Insecure (HTTP) Endpoints
```

Comment:
We didn't take this patch, so leave out this section.  The rest is ok though.

Reply:
I removed this section